Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mesosphere/mesos:0.21.1-1.1.ubuntu1404
+
+RUN apt-get update && apt-get -y install gettext
+
+ENV HDFS_MESOS_VERSION 0.1.1
+
+COPY hdfs-mesos-$HDFS_MESOS_VERSION.tgz /hdfs/
+RUN cd /hdfs && tar xfz hdfs-mesos-$HDFS_MESOS_VERSION.tgz
+
+ENV MESOS_NATIVE_JAVA_LIBRARY /usr/local/lib/libmesos.so
+ENV MESOS_NATIVE_LIBRARY /usr/local/lib/libmesos.so
+
+ADD conf/mesos-site.xml /tmp/mesos-site.xml
+
+WORKDIR /hdfs/hdfs-mesos-$HDFS_MESOS_VERSION
+
+CMD ./bin/hdfs-mesos
```

Comment:
Please use the fully qualified path here to ensure that and potential change to workdir doesn't impact the ability to run the script.